Support loading YTDL_OPTIONS from file

ArabCoders opened the pull request:
This PR implements a way to load yt-dlp custom options from file using the same environment variable name, i also updated the README.md to reflect the changes.

This PR closes #210

alexta69 commented on app/main.py:
```diff
@@ -12,6 +12,8 @@
 from ytdl import DownloadQueueNotifier, DownloadQueue
 
 log = logging.getLogger('main')
+if __name__ == '__main__':
```
Why has this moved here? There's no reason to move it...

ArabCoders replied:
Because the call to Config comes before the logging level is set as such if you use debug level it wont show up. at least from my testing.

ArabCoders replied:
@alexta69  I can move it back but the logging in the Config class will have to be changed to INFO or removed as it's won't show up otherwise.

alexta69 commented:
@ArabCoders I reworked the PR a bit -- I was a bit concerned about trying to interpret the same env variable in two different ways (JSON and file path) so I switched to a separate env variable for specifying the file. I hope it works well for you. Thanks!

ArabCoders commented:
> @ArabCoders I reworked the PR a bit -- I was a bit concerned about trying to interpret the same env variable in two different ways (JSON and file path) so I switched to a separate env variable for specifying the file. I hope it works well for you. Thanks!

Oh no worries thanks for merging the support for it. If you have a bit of time could you please take look at https://github.com/alexta69/metube/pull/310 i posted screenshot of an edge case that happens when download fail.  

Thank You.
